src/components/Result.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { Button, Jumbotron } from 'react-bootstrap';
+
+const Result = ({ score, NumOfQuestions }) => {
+  return (
+    <div>
+      <Jumbotron>
+        <h2>
+          You have got {score} out of {NumOfQuestions}
+        </h2>
+      </Jumbotron>
+      <Button
+        className="mr-2 mb-4"
+        variant="secondary"
+        href="quiz"
+      >
+        Try again
+        </Button>
+      <Button
+        className="mr-2 mb-4"
```

Comment:
We don't need these classnames — we don't need a margin on the bottom because the `Try again` button already has this, and we don't need a margin on the right because there's nothing on the right to make a space for

Reply:
Done.